src/hooks/validation/MultiSignerValidatorHook.sol

```diff
@@ -8,10 +8,36 @@ import {Call, CallLib} from "../../libraries/CallLib.sol";
 import {IValidationHook} from "../../interfaces/IValidationHook.sol";
 import {AccountKeyHash, AccountKeyHashLib} from "../shared/AccountKeyHashLib.sol";
 
+/// @notice A transient storage library for hook data
+/// TODO: This library can be deleted when we have the transient keyword support in solidity.
+library HookDataTransientStorage {
```

Comment:
This contract is subject to change in the future, just wanted to get something rough working without hookData